gvsbuild/projects.py

```diff
@@ -409,6 +409,30 @@ def build(self):
         Meson.build(self, meson_params='-Ddocs=false')
         self.install(r'.\COPYING share\doc\fribidi')
 
+@project_add
+class Project_pdfium(GClientRepo, Project):
+    def __init__(self):
+        Project.__init__(self,
+            'pdfium',
+            repo_url = 'https://pdfium.googlesource.com/pdfium.git',
+            fetch_submodules = False,
+            tag = 'df97b13e53d1b96151fa119e3e94db19f5df097b',
+            dependencies = ['depot', 'python', 'ninja'],
+            patches = ['shared_library.patch', 'relative_includes.patch'],
+            subproject_patches = {'build': ['rc_compiler.patch']},
+            )
+
+    def build(self):
+        self.exec_vs(r'cp ..\resources.rc .')
+        self.exec_vs(r'cp ..\args.gn out')
+        if self.builder.x86:
+            self.exec_vs(r'echo target_cpu="x86" >> out\args.gn')
+        self.exec_vs(r'gn gen out')
+        self.exec_vs(r'ninja -C out pdfium')
+
+        self.install_dir(r'public', r'include\pdfium')
+        self.install(r'out\pdfium.dll.lib', r'lib')
```

Comment:
I think we want this to be named pdfium.lib for consistency with other libraries